suite-common/wallet-config/src/utils.ts

```diff
@@ -69,3 +62,6 @@ export const getNetworkFeatures = (symbol: NetworkSymbol) =>
     networks[symbol]?.features as unknown as NetworkFeature;
 
 export const getCoingeckoId = (symbol: NetworkSymbol) => networks[symbol]?.coingeckoId;
+
+export const isNetworkSymbol = (symbol: NetworkSymbol | string): symbol is NetworkSymbol =>
+    networks.hasOwnProperty(symbol);
```

Comment:
I changed this from `symbol in networks` to `hasOwnProperty`, because for example `'toString'` shouldn't be evaluated as a valid network symbol :innocent: 

:thought_balloon: IMHO `in` is fine if we are checking a specific key which surely isn't inherited from prototype, like `'btc' in networks`, but a function that checks any string in general should use `hasOwnProperty` to exclude inherited properties.